Atala PRISM/2.9.md

```diff
@@ -0,0 +1,68 @@
+
+## Components
+
+### Platform
+
+* [Open Enterprise Agent v1.25.0](https://github.com/hyperledger-labs/open-enterprise-agent/releases/tag/prism-agent-v1.25.0)
+* [PRISM Node v2.2.1](https://github.com/input-output-hk/atala-prism/releases/tag/v2.2.1)
+* [PRISM Mediator v0.10.2](https://github.com/input-output-hk/atala-prism-mediator/releases/tag/prism-mediator-v0.10.2)
+
+### SDKs
+
+* [Apollo Crypto Library v1.2.10](https://github.com/input-output-hk/atala-prism-apollo/releases/tag/v1.2.10)
+* [Wallet SDK Swift v4.0.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift/releases/tag/4.0.0)
+* [Wallet SDK TypeScript v4.0.2](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts/releases/tag/v4.0.1)
+* [Wallet SDK KMM v2.7.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/releases/tag/v2.7.0)
+
+### Documentation
+
+* [Documentation Portal v1.76.0](https://github.com/input-output-hk/atala-prism-docs/releases/tag/v1.76.0)
+
+## Changelog
+
+**Updated components:**
+
+- Open Enterprise Agent: v1.21.0 -> v1.25.0
+- Documentation Portal: v1.69.0 -> v1.76.0
+- Apollo Crypto Library: v1.1.1 -> v1.2.10
+- Wallet SDK KMM: v2.6.0 -> v2.7.0
+- Wallet SDK TypeScript: v3.1.0 -> v4.0.2
+- PRISM Mediator: no changes
+- Wallet SDK Swift: v3.6.0 -> v4.0.0
+- PRISM Node: no changes
+
+**New Features:**
+
+- **Open Enterprise Agent**:
+  - Added JWT auth support for agent-admin role
+  - Upgraded ZIO http client to improve performance
+  - Improved documentation of the OpenAPI specification
+- **Documentation Portal**:
+  - Build SDK TypeScript documentation using GitHub submodules
+  - Added sidebar for SDK TypeScript
+  - Added JWT role to documentation authentication 
+- **Apollo Crypto Library**:
+  - Fixed `Crypto` object undefined
+- **Wallet SDK TypeScript**: 
+  - **BREAKING CHANGES**:
+    - PrivateKey.index type now is `number | undefined` PrivateKey.index was returning a `string`. It's possible where that's being used (ie Pluto) will need to handle the new `number | undefined` type
+    - The SDK is now exported as default, in order to import it use `import sdk from '@atala/prism-wallet-sdk'` vs `import * as sdk from '@atala/prism-wallet-sdk'`. Browsers now also don't need to build complex webpack configuration, check the demos for more info.
+    - Pollux requires a new function `createPresentationProof` to handle creating a Presentation for the relevant Request and Credential.
+  - Added `Apollo` integration
+  - Added `ExportableKey` protocol and implementation
+  - Added hybrid `browser-nodejs` as `commonjs` and `module` dependency
+  - Fixes JTI field recovery, allowing to regenerate the original JWT
+- **Wallet SDK KMM**:
+  - Added the capability ro receive and process presentation request
+  - Fixed DIDCommWrapper which was crashing when body is empty string
+- **Wallet SDK Swift**:
+  - **BREAKING CHANGES**:
+    - Updated `Apollo` public interface to be more in line with Cryptographic abstraction
```

Comment:
We could merge with the line below "Added `Apollo` integration" as it is confusing to indicate twice the same.
So I would suggest "Updated public interface to integrate Cryptographic abstraction module Apollo"